Correct image examples (#88)
AdobeDocs/commerce-frontend-core: src/pages/guide/translations/practice.md

ExampleCorp created a custom `orange` theme that inherits from the Blank theme. When customizing their theme, they want to rephrase certain strings used in the Blank theme and modules for the default locale.

> The following image shows a page with the default string values.
> 
> ![Product page where the customized Compare string is displayed](../../_images/frontend/fdg_trans_bag.png)

Author: ```suggestion
The following image shows a  product view with default string values for .

![Product page with default string values](../../_images/frontend/fdg_trans_bag.png)
```

> ExampleCorp wants to customize these values to make the following changes:

Reviewer: To avoid "want", perhaps "prefers" ?

> ExampleCorp wants to customize these values to make the following changes:

Author: ```suggestion
ExampleCorp has requirements to change the default strings to the following values:
```

For example:

> ![Product page where the Add to Compare string is displayed](../../_images/frontend/fdg_translations_bag2.png)

Author: ```suggestion
![Product page after applying theme with custom strings](../../_images/frontend/fdg_translations_bag2.png)
```
